primefaces/src/main/resources/META-INF/resources/primefaces/jquery/jquery.ui.pfextensions.css

```diff
@@ -3,22 +3,21 @@
     padding: 0.3em;
 }
 
-.ui-icon { 
-    display: block; 
-    vertical-align: baseline;
+.ui-icon {
+    display: block;
     margin-top: 0px;
 }
 
-/* https://stackoverflow.com/a/62109988/502366 */
+/* Modern override for all browsers and screenreaders */
 .ui-helper-hidden-accessible {
```

Comment:
Thank you for the quick PR!

As long as they aren't actually added by jQuery UI (I'm not sure if they are or not), I don't think you need to "unset" the values for border, clip, height, margin, overflow, padding and width. Also, I do think that you need to keep the position at "absolute" otherwise in our case at least, the layout was a little broken (they keep occupying space).

Reply:
Yes these are set by jQuery UI so we need to unset them as you did.